### ISSUE TITLE
[iOS] SetApplyShadow doesn't work on iPad

### ISSUE THREAD

**Issue by bpater-tp** [xamarin/Xamarin.Forms#11168]
### Description

The new feature allowing to put shadow on Detail Page does work nicely on iPhones. On iPad however it is not working at all.

### Steps to Reproduce

1. Download Platform Specifics demo app
2. From iOS list select "MasterDetailPage Shadow"
3. Using swipe from left screen edge gesture show Master Page (aka sidebar)

### Expected Behavior

On both device types it should behave in the same way - make Detail page darker when Master Page is visible if ApplyShadow is set to true.

### Actual Behavior

Currently there's different behaviour on iPhone and iPad:
- on iPhone it makes Detail Page darker when Master Page is visible
- on iPad nothing changes to Detail Page

### Basic Information

- Version with issue: 4.6.x or higher
- Last known good version: none
- IDE: Visual Studio for Mac v8.6.4
- Platform Target Frameworks: 
  - iOS: 13.18.2.1

### Screenshots

![Simulator Screen Shot - iPhone 11 - 2020-06-23 at 08 47 43](https://user-images.githubusercontent.com/19667183/85369955-4c36bc00-b52e-11ea-8403-5e11ed2f4372.png)
![Simulator Screen Shot - iPad (7th generation) - 2020-06-23 at 08 47 14](https://user-images.githubusercontent.com/19667183/85369961-4e007f80-b52e-11ea-909b-9a7dee4a1c48.png)

### Reproduction Link

https://docs.microsoft.com/en-us/samples/xamarin/xamarin-forms-samples/userinterface-platformspecifics/

### Workaround

None known

**Comment by zizusoft** [comment]
Any news on this? It is still not working in Xamarin Forms v5

**Comment by bpater-tp** [comment]
yeah, this feature is basically unusable without proper iPad implementation

**Comment by zizusoft** [comment]
It was fixed for iPad in 4.6.0.494-pre2 apparently

**Comment by bpater-tp** [comment]
@zizusoft no, it is not fixed. just checked.
